keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -483,18 +510,21 @@ Put more succinctly:
 CSI clients that will need updating are kubelet, external-provisioner,
 external-attacher, and external-resizer.
 
+### Supporting In-Tree Drivers
+
+In-tree storage drivers implement the [`PersistentVolumePlugin`] interface which
```

Comment:
I am a little hesitant on supporting in-tree drivers because this adds more testing requirements on each plugin owner. And many plugins have already GA-ed their csi migration features.

Reply:
Ok, I can remove this.